test/boost/sstable_datafile_test.cc

```diff
@@ -1768,9 +1768,10 @@ static shared_sstable sstable_for_overlapping_test(test_env& env, const schema_p
 
 // ranges: [a,b] and [c,d]
 // returns true if token ranges overlap.
-static bool key_range_overlaps(sstring a, sstring b, sstring c, sstring d) {
-    auto range1 = create_token_range_from_keys(a, b);
-    auto range2 = create_token_range_from_keys(c, d);
+static bool key_range_overlaps(column_family_for_tests& cf, sstring a, sstring b, sstring c, sstring d) {
```

Comment:
nitpick (feel free to ignore) - you didn't need cf here, schema& s would have been enough.

Reply:
I agree. A partitioner would be even more specific but I just wanted to reduce the size of the change. If I take schema I would have to place cf->schema() in all the places the method is called and there's quite a few of them. I didn't want to repeat myself.